test/unit/dialects/postgres/query-generator.test.js

```diff
@@ -897,5 +897,53 @@ if (dialect.match(/^postgres/)) {
         });
       });
     });
+
+    describe('JSON', function () {
```

Comment:
OPTIONAL: 
If you are a completely awesome human being, please rewrite these as unit tests ,along the lines of https://github.com/sequelize/sequelize/blob/master/test/unit/sql/json.test.js